src/psyclone/psyir/backend/fortran.py

```diff
@@ -216,12 +257,14 @@ def container_node(self, node):
 
         :raises VisitorError: if the name attribute of the supplied \
         node is empty or None.
+        :raises VisitorError: if any of the children of the supplied \
+        Container node are not KernelSchedules.
```

Comment:
s/node/nodes/ and please indent this second line a bit.

Reply:
I think it should be singular as we're talking about the Container node that is passed into this routine.
I'm also not sure what you want in terms of indentation. I always line-up multiple lines this way, although I know others don't, so it would be weird to just do this one. For the record I don't indent because a) emacs doesn't when formatting and b) I don't like having multiple lines of text pushed over to the rhs - which is what can sometimes happen, although I accept that your 'indent a bit' suggestion gets round this. So far we've informally gone down the line of allowing people to format multiple lines that describe the interface in docstrings in whatever way they prefer. However, if we want consistency then formatting could be discussed at our next telco?

Reply:
Oh yes, I don't know why I thought it should be plural. I really prefer a bit of indentation as I find it easier to read but I'm not sure I have the energy for a full on discussion of it. I'll let it go.

Reply:
I believe our coding style covers that case already:
```
     If a parameter description, type, return value or exception is continued
     to the next line, there must be a '\\\\' continuation symbol at the end of
     each line. Align each continued line with the column at which the
     description begins on the previous line. If this would create lines that
     are too short then the first continued line may be indented less ...
```
Maybe (I didn't try, I moved away from emacs) https://emacs.stackexchange.com/questions/24085/how-to-get-emacs-to-automatically-insert-block-comment-prefixes
helps?